docs/releases/handling-releases.md

```diff
@@ -199,6 +199,23 @@ Use previous posts for inspiration. If the release contains new features, or API
 
 _Outcome:_ __There's a public release announcement, with clear info about what's new, roughly within a day of actual release.__
 
+#### Update user-facing documentation as needed
+
+When the plugin is released, ensure user-facing documentation is kept up to date with new blocks and compatibility information. The dev team should update documents in collaboration with support team and WooCommerce docs guild.
+
+In particular, please review and update as needed:
+
+- Are there any new blocks in this release? Ensure they have adequate user documentation.
+- Ensure any major improvements or changes are documented.
```

Comment:
where should this information live?

Reply:
By "this information" do you mean:

1. The hint/process that the docs should be updated as part of the release.
2. The information about the actual improvements / changes to the plugin.

Either way, a good question – maybe add this to team async meeting thread (and/or sprint retro) for wider discussion.

Here's my thoughts on the two above items:

1. I think there should be a reminder to update the docs in the release process, especially for the version/compatibility info. However I don't think this should be the only place we consider this – should be keeping it in mind over sprint process.
2. New blocks and features should documented on http://docs.woocommerce.com.

All that said, the release process doc is getting quite long. Maybe we could move some of this material to team handbook (P2) and keep this doc as a lean checklist.